resources/css/app.css

```
@@ -146,6 +149,7 @@
     --sidebar-accent-foreground: oklch(0.985 0 0);
     --sidebar-border: oklch(0.269 0 0);
     --sidebar-ring: oklch(0.439 0 0);
+    --footer-background: HSL(45, 7%, 7%);
```

Should be named just footer and should have foreground color for text.
```suggestion
    --footer: HSL(45, 7%, 7%);
    --footer-foreground: HSL(0, 0%, 80%)
```